src/main/java/com/group3/backend/ui/controller/MedicationDetailsController.java

```diff
@@ -0,0 +1,27 @@
+package com.group3.backend.ui.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+import com.group3.backend.ui.model.request.MedicationDetailsModel;
+import com.group3.backend.ui.model.response.MedicationDetailsRepository;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/api")
+public class MedicationDetailsController {
+	@Autowired
+	private MedicationDetailsRepository medicationDetailsRepo;
+
+	@GetMapping("/medications")
+	public List<MedicationDetailsModel> getMedication()
+	{
+		return medicationDetailsRepo.findAll();
```

Comment:
Bad practice to expose all fields of the entity normally in case there's sensitive data there. For instance, the care workers if we returned the entity we give away their id, email and password hash. That's why we have response classes. In this case it doesn't matter too much as it's only the Medication

Reply:
Sure, I will work to fix it by using the response class.